### PR TITLE
Fix pagination for list_organizational_units_for_parent

### PR DESCRIPTION
**Pull request by lvthillo**
See #58 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: amplify/backend/function/teamgetOUs/src/index.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def getOUs(id):
         )
         results = response["OrganizationalUnits"]
         while "NextToken" in response:
-            response = client.list_accounts(NextToken=response["NextToken"])
+            response = client.list_organizational_units_for_parent(NextToken=response["NextToken"])
```
Comment (andyt10) [inline amplify/backend/function/teamgetOUs/src/index.py:19]: I think you also need  `ParentId=id` in params. 
I tried without and it and got a param exception:

```
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Missing required parameter in input: "ParentId"
```

Comment (lvthillo) [reply amplify/backend/function/teamgetOUs/src/index.py]: Yes indeed; Thanks for checking again. I can't test this properly in my environment. I'll try to setup another env on which I can test this properly. For now, if you copy the code in this file to your lambda, it works fine for you?

**Comment by lvthillo** [comment]
FYI found a way to test this without creating many OUs. By forcing a max of 3 results we will go in the while loop and use NextToken. This seems to work now. Thanks for your help @andyt10.
```
        response = client.list_organizational_units_for_parent(
            ParentId=id, MaxResults=3
        )
```

**Comment by tawoyinfa** [comment]
Thanks for supporting this @lvthillo and @andyt10

**Pull request merged**